webui/src/models/cluster/suggestions/index.ts

```diff
@@ -0,0 +1,132 @@
+import { combine } from 'effector';
+
+import type {
+  DisableServerSuggestion,
+  EditServerInput,
+  ForceApplySuggestion,
+  RefineUriSuggestion,
+  RestartReplicationSuggestion,
+} from 'src/generated/graphql-typing-ts';
+import { app } from 'src/models';
+
+import type { CheckedServers, ForceApplySuggestionByReason, SuggestionsPanelsVisibility } from './types';
+
+// events
+export const advertiseURIApplyClick = app.domain.createEvent<unknown>('advertise URI apply click');
+export const advertiseURIDetailsClick = app.domain.createEvent<unknown>('advertise URI details click');
+export const disableServersApplyClick = app.domain.createEvent<unknown>('disable servers apply click');
+export const disableServersDetailsClick = app.domain.createEvent<unknown>('disable servers details click');
+export const restartReplicationsApplyClick = app.domain.createEvent<unknown>('restart replications apply click');
+export const restartReplicationsDetailsClick = app.domain.createEvent<unknown>('restart replications details click');
+export const forceApplyConfApplyClick = app.domain.createEvent<unknown>('force apply config apply click');
+export const forceApplyConfDetailsClick = app.domain.createEvent<unknown>('force apply config details click');
+export const forceApplyInstanceCheck = app.domain.createEvent<string>('force apply check instance');
+export const forceApplyReasonCheck = app.domain.createEvent<string>('force apply check instances with same reason');
+export const forceApplyReasonUncheck = app.domain.createEvent<string>('force apply uncheck instances with same reason');
+export const detailsClose = app.domain.createEvent('details modal close');
```

Comment:
usually events were named with a postfix "Event"

Reply:
This is the old functionality. I left it as is.